templates/usr/local/sbin/secure_owncloud.sh.j2

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# This script sets the recommended modes and ownership for your ownCloud 
+# directories and files
+# https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label
+
+# {{ ansible_managed }}
+
+[[ "$UID" != 0 ]] && { echo "You must be root"; exit 1; }
+
+ocpath='{{ owncloud_home }}'
+htuser='{{ owncloud_user }}'
+htgroup='{{ owncloud_group }}'
```

Comment:
As this script is already templates by Ansible, I think you can drop the extra variables and use `{{ owncloud_home }}` and so on directly.